packages/react/src/components/Button/Button.module.css

```
@@ -92,7 +92,7 @@
 }
 
 .button.onlyIcon {
-  padding: 0.5rem;
+  padding: 0.3125rem;
```

Foreslår å bytte ut dette med en utregning, så slipper vi å måtte endre på dette hvis noen av de andre verdiene skulle endre seg. Da er det også tydelig hvor verdien kommer fra.
```suggestion
  padding: calc((var(--button-size) - var(--icon-size)) / 2 - var(--component-button-border_width-default));
```